bc_obps/service/contact_service.py

```diff
@@ -37,6 +40,34 @@ def get_if_authorized(cls, user_guid: UUID, contact_id: int) -> Optional[Contact
             raise Exception(UNAUTHORIZED_MESSAGE)
         return contact
 
+    @classmethod
+    def get_with_places_assigned(cls, user_guid: UUID, contact_id: int) -> Optional[Contact]:
+        contact = cls.get_if_authorized(user_guid, contact_id)
+
+        places_assigned = []
+        if contact:
```

Comment:
I had to add this line to make mypy happy because it didn't seem to get that on line 45, if there's no contact, we raise an exception. It's going to make sonar sad though

Reply:
stuck between a rock and sonarcloud...

Reply:
Pulling out the formatting logic worked for sonarcloud

Reply:
This ended up changing per Zoey/Patricia, so not an issue anymore